### ISSUE TITLE
Input Normalization

### ISSUE THREAD

**Issue by JunzheJosephZhu** [JusperLee/Dual-Path-RNN-Pytorch#16]
I'm not sure if my mixing is exactly same as yours, but does your torchaudio read the wav files to int(value is typically around a couple hundred) or to float values between [-1, 1]?

I started with scipy which loads to int, and it caused loss going to NaN at a point. So I switched to librosa which loads to float

**Comment by JusperLee** [comment]
float values between [-1, 1]. if you started with scipy which loads to int, you will norm audio between [-1,1]

**Comment by JunzheJosephZhu** [comment]
how long did it take for you to train? I'm using 4x1080Ti and each epoch takes around 2 hrs. I use wsj0-2mix, and take 4s windows with 2s skip size, with 2s least length. Is this speed normal?

**Comment by JunzheJosephZhu** [comment]
Also, just to confirm, norming between [-1, 1] is just dividing int by 2**15, right? Or is it minmax normalization?
